saleor/tax/models.py

```diff
@@ -54,7 +54,11 @@ class TaxConfiguration(ModelWithMetadata):
     )
     charge_taxes = models.BooleanField(default=True)
     tax_calculation_strategy = models.CharField(
-        max_length=20, choices=TaxCalculationStrategy.CHOICES, blank=True, null=True
+        max_length=20,
+        choices=TaxCalculationStrategy.CHOICES,
+        blank=True,
```

Comment:
If we have a default value, do we still need `blank=True` and `null=True`? If you would make migration to overwrite `null` values and set `FLAT_RATES` then you don't need:
https://github.com/saleor/saleor/pull/11375/files#diff-82a64a5b6c5d44e2c458ee0a81b3f9145c08091533541300e35ddd4bbb9a217aR73
(And migration from `None` to `FLAT_RATES` should be safe, as we already assume that `FLAT_RATES` should be assigned in case of missing value.
Right?